### PR TITLE
Add private and public fields to wishlist

### PR DESCRIPTION
**Pull request by riteshshrv**
Adds two booleans public and private to implement 'Share' feature

### CODE REVIEW

File: wishlist.xml [hunk 1]
```diff
@@ -1,9 +1,15 @@
 <tryton>
     <data>
         <record model="ir.ui.view" id="wishlist_form">
-        <field name="model">nereid.user</field>
-        <field name="inherit" ref="nereid.nereid_user_form2"/>
-        <field name="name">wishlist_form</field>
+	        <field name="model">nereid.user</field>
+	        <field name="inherit" ref="nereid.nereid_user_form2"/>
+	        <field name="name">wishlist_form</field>
```
Comment (priyankarani) [inline wishlist.xml]: revert this change

File: wishlist.py [hunk 2]
```diff
@@ -62,6 +62,9 @@ class Wishlist(ModelSQL, ModelView):
     """
     __name__ = "wishlist.wishlist"
 
+    public = fields.Boolean('Public', select=True)
+    private = fields.Boolean('Private', select=True)
```
Comment (priyankarani) [inline wishlist.py]: can you rename fields to is_public and is_private so it tells its boolean field.
No need to change field string.

**Comment by priyankarani** [comment]
change commit message to "Add private and public fields to wishlist"

File: view/wishlist_view_form.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<form string="Wishlist" col="6">
+    <label name="name"/>
+    <field name="name" xexpand="1"/>
+    <group col="20" colspan="2" id="checkboxes">
+            <label name="is_public"/>
```
Comment (tarunbhardwaj) [inline view/wishlist_view_form.xml]: fix indentation

File: wishlist.py [hunk 4]
```diff
@@ -62,6 +62,9 @@ class Wishlist(ModelSQL, ModelView):
     """
     __name__ = "wishlist.wishlist"
 
+    is_public = fields.Boolean('Public', select=True)
+    is_private = fields.Boolean('Private', select=True)
```
Comment (tarunbhardwaj) [inline wishlist.py]: not required as it is default behaviour
